### DISCUSSION TITLE
Encoder errror

### DISCUSSION THREAD

**Discussion by luke2023** [pschatzmann/arduino-audio-tools#400] [Q&A] [answered]
hello, I replaced the wavEncoder with mp3encoder lame and it there is a error here,is this normal?thanks 
![194583724-ae39fcc5-a463-4072-936c-ca8cf6a65ecb](https://user-images.githubusercontent.com/110216618/194714841-a43970cd-e72d-4208-a742-591de9d11dc7.jpeg)

**Comment by pschatzmann** [comment, marked as answer]
Yes that's normal if you did not install the library and you did not add the necessary include.
Did you read  the related [Wiki](https://github.com/pschatzmann/arduino-audio-tools/wiki/Encoding-and-Decoding-of-Audio) ?
